NumLib/ODESolver/TimeDiscretizedODESystem.h

```diff
@@ -81,9 +81,7 @@ class TimeDiscretizedODESystem<ODESystemTag::FirstOrderImplicitQuasilinear,
 
     ~TimeDiscretizedODESystem();
 
-    void assembleResidualNewton(const GlobalVector& x_new_timestep) override;
-
-    void assembleJacobian(const GlobalVector& x_new_timestep) override;
+    void assemble(const GlobalVector& x_new_timestep) override;
```

Comment:
In the old version it was clear from the function's name, what it assembles. Maybe adding some comments on what is assembled now would be helpful.